UpdateRCConfiguration/__tests__/updateRCConfiguration.test.ts

```diff
@@ -0,0 +1,92 @@
+import * as E from "fp-ts/lib/Either";
+import * as TE from "fp-ts/lib/TaskEither";
+import * as O from "fp-ts/lib/Option";
+
+import {
+  handleEmptyConfiguration,
+  handleGetLastRCConfigurationVersion,
+  handleUpsert,
+  isUserAllowedToUpdateConfiguration
+} from "../handler";
+import {
+  aRemoteContentConfiguration,
+  findLastVersionMock,
+  rccModelMock,
+  upsertConfigurationMock
+} from "../../__mocks__/remote-content";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+describe("isUserAllowedToUpdateConfiguration", () => {
+  test("should return a left if the userId is not equal to the userId of the configuration", async () => {
+    const r = await isUserAllowedToUpdateConfiguration(
+      "aDifferentUserId" as NonEmptyString
+    )(aRemoteContentConfiguration)();
+    expect(E.isLeft(r)).toBeTruthy();
```

Comment:
I would replace `tobeTruthy` with `toBe(true)` since `E.isLeft` returns a boolean value

Reply:
[a6d6f43](https://github.com/pagopa/io-functions-service-messages/pull/47/commits/a6d6f436345a0acc70a051b95989980b4918fcb3)

Reply:
I'd check even the other `tobeTruthy` to be sure.